lib/git-lfs-s3/application.rb

```diff
@@ -101,7 +101,7 @@ def public_read_grant
         status 404
       end
       if settings.public_server and settings.ceph_s3
-        if object.acl.grants.include?(public_read_grant)
+        if not object.acl.grants.include?(public_read_grant)
```

Comment:
I don't think we tested push other than pushing afwdata - which was a one off. I changed permissions of the keys for afwdata.